microsoft-365/lighthouse/m365-lighthouse-manage-tenant-list.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Manage your tenant list"
+title: "Manage Lighthouse tenant list"
```

Comment:
I fixed this in a different PR. I kept the word "your" because it's specifically referring to the tenant list owned by the MSP technician who's reading the article. However, I did add the product name (we need to use the full name of Microsoft 365 Lighthouse in titles and H1 headings to differentiate from the Azure Lighthouse product.)